website/docs/r/org.html.markdown

```diff
@@ -47,6 +58,26 @@ The following arguments are supported:
 * `stored_vm_quota` - (Optional) - Maximum number of virtual machines in vApps or vApp templates that can be stored in an undeployed state by a member of this organization. Default is unlimited (0)
 * `can_publish_catalogs` - (Optional) - True if this organization is allowed to share catalogs. Default is `true`.
 * `delay_after_power_on_seconds` - (Optional) - Specifies this organization's default for virtual machine boot delay after power on. Default is `0`.
+* `vapp_lease` - (Optional; *v2.7*) - Defines lease parameters for vApps created in this organization. See [vApp Lease](#vapp-lease) below for details. 
+* `vapp_template_lease` - (Optional; *v2.7*) - Defines lease parameters for vApp templates created in this organization. See [vApp Template Lease](#vapp-template-lease) below for details.
+
+<a id="vapp-lease"></a>
+## vApp Lease
+
+The `vapp_lease` section contains lease parameters for vApps created in the current organization, as defined below:
+
+* `maximum_runtime_lease_in_sec` - (Optional) - How long vApps can run before they are automatically stopped (in seconds). Minimum 3600. - Default is 604800 (7 days)
```

Comment:
The value for 'Never Expires' should be documented

Reply:
Thanks for looking at it. Good call.
I need to add a call for this item, as the current API can't set the `never expires` value.

Reply:
Support for 'Never Expired' added